e2e/tests/ui-driven/src/helpers/types.ts

```diff
@@ -0,0 +1,23 @@
+import { CoreDomainClient } from "@opensystemslab/planx-core";
+import { User } from "@opensystemslab/planx-core/dist/types";
+
+type NewTeam = Parameters<CoreDomainClient["team"]["create"]>[0];
+
+export interface Flow {
+  id?: string;
+  publishedId?: number;
+  slug: string;
+  name: string;
+  data?: object;
+}
+
+export interface TestContext {
```

Comment:
nit: I'm not sure if `TestContext` is really much clearer or better than `Context` here when we're entirely within Playwright tests.

Not a strongly held opinion - was there a motivation for this change? Classes with `BrowserContext` maybe?

Reply:
I can't fully remember why, but it did come out of some frustration with the generalised context. Might have been to help express that as a Type it was generated within the Test, rather than bringing it in from external Types. If that makes sense?